src/Pages/Home/Home.js

```diff
@@ -164,11 +43,26 @@ const brands=[
     imgUrl:"./Assets/Zara-Logo.png"
   }
 ]
+
 function Home() {
+  const [newArrivals,setNewArrivals] =useState([]);
+  useEffect(()=>{
+    
+      const fetchNewArrivals=async()=>{
+
+      const response=await fetchData('/products/new');
```

Comment:
do you need to use new arrivals data in other component in home?
if not please move the logic for fetching the data inside NewArrivals component.

and handle loading and errors properly.